tensorflow_model_optimization/python/core/quantization/keras/graph_transformations/model_transformer.py

```diff
@@ -424,7 +424,7 @@ def _weight_name(name):
     Returns:
       Extracted weight name.
     """
-    return name.split('/')[-1]
```

Comment:
Could you add a unit test for this change so we don't make the same mistake going forward?